### PR TITLE
Expose bits_per_coded_sample on VideoCodecContext

### PR DESCRIPTION
**Pull request by rvanlaar**
qtrle decoder needs `bits_per_coded_sample` to be set when decoding a packet.

### CODE REVIEW

**Comment by jlaine** [comment]
Can we have a unit test which exercises the getter and setter please?

File: av/video/codeccontext.pyx [hunk 1]
```diff
@@ -92,6 +92,14 @@ cdef class VideoCodecContext(CodecContext):
             self.ptr.height = value
             self._build_format()
 
+    property bits_per_coded_sample:
+        def __get__(self):
```
Comment (jlaine) [inline av/video/codeccontext.pyx:96]: This needs a docstring

File: av/video/codeccontext.pyx [hunk 2]
```diff
@@ -92,6 +92,14 @@ cdef class VideoCodecContext(CodecContext):
...
+            return self.ptr.bits_per_coded_sample
+
+        def __set__(self, unsigned int value):
```
Comment (jlaine) [inline av/video/codeccontext.pyx:99]: Why an *unsigned* int?

**Comment by WyattBlue** [comment]
In addition to jlaine's comments, this needs to be redone with the new style property attribute and it needs the type hint.

**Pull request merged**